README.md

```diff
@@ -153,15 +153,51 @@ cd myproject
 $GOPATH/bin/apikit validate doc/myproject.yaml
 ```
 
-### Generate API server and client
+### Generate API server, client and mock client
 
-The `apikit generate <api.yaml> <dest.dir> <package>` command generates the client and server API based on an OpenAPIv2 definition. Use `--only-client` if the server component and `--only-server` if the client component is not needed.
+The `apikit generate <api.yaml> <dest.dir> <package> <flags>` 
+command generates the client, mock client and server API based on an OpenAPIv2 definition.
+
+* (Optional) Use flag `--only-client` to only generate the server component should be generated.
+* (Optional) Use flag `--only-server` to only generate the client component should be generated.
```

Comment:
@donutloop Please correct the description of the `--only-server` flag to the following.

'(Optional) Use flag `--only-server` to only generate the server components'

Reply:
Mmh yeah

Reply:
fixed